dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -73,7 +73,8 @@ public Pattern getParameterPattern() {
                 DbParameterAccessor dbp = new DbParameterAccessor(columnName,
                         Direction.INPUT,
                         typeMapper.getJDBCSQLTypeForDBType(dataType),
-                        getJavaClass(dataType), position++);
+                        getJavaClass(dataType), position++,
+                        typeSpecifiers);
```

Comment:
I can pull some stuff back into `AbstractDbEnvironment` but the adapters don't create `DbParameterAccessor`s in a uniform way. Some use `DbParameterAccessorsMapBuilder` to build up a `Map` and the explode that into a `Map<String, DbParameterAccessor>`.